impl/src/main/java/com/okta/sdk/impl/oauth2/AccessTokenRetrieverServiceImpl.java

```diff
@@ -128,23 +128,26 @@ public OAuth2AccessToken getOAuth2AccessToken() throws IOException, InvalidKeyEx
      * @return signed JWT string
      * @throws InvalidKeyException
      * @throws IOException
+     * @throws IllegalArgumentException if <code>accessTokenTtl</code> property is missing or negative
      */
     String createSignedJWT() throws InvalidKeyException, IOException {
+        Duration accessTokenTtl = tokenClientConfiguration.getAccessTokenTtl();
```

Comment:
We should validate for `exp` parameter for not being more than 1 hour into the future.